README.md

```diff
@@ -14,91 +15,91 @@
 <p align="center">
   <img src="./assets/preview.webp"/>
 </p>
-
-## Content
-
-1. [Themes](#themes)
-1. [Installation](#installation)
-1. [Overview](#overview)
-1. [Configuration options](#configuration-options)
-   1. [Window](#window)
-   1. [Pane](#pane)
-   1. [Status Line](#status-line)
-   1. [Using the theme's built-in status modules](#using-the-themes-built-in-status-modules)
-1. [Configuration Examples](#configuration-examples)
-   1. [Config 1](#config-1)
-   1. [Config 2](#config-2)
-   1. [Config 3](#config-3)
+<!-- markdownlint-enable -->
 
 ## Themes
 
-- 🌻 [Latte](./themes/catppuccin_latte_tmux.conf)
-- 🪴 [Frappé](./themes/catppuccin_frappe_tmux.conf)
-- 🌺 [Macchiato](./themes/catppuccin_macchiato_tmux.conf)
-- 🌿 [Mocha](./themes/catppuccin_mocha_tmux.conf)
+<details>
+<summary>🌻 Latte</summary>
 
-## Installation
+![Latte Flavor Preview](./assets/latte.webp)
 
-> [!IMPORTANT]
-> The plugin was re-written from bash to use tmux's native configuration
-> language. This occurred after `v0.3.0`. Old configurations will
-> need to be updated. Please consider if the upgrade is worth the tradeoff.
+</details>
+<details>
+<summary>🪴 Frappé</summary>
 
-In order to have the icons displayed correctly please use/update your favorite [patched font](https://www.nerdfonts.com/font-downloads).
-If you do not have a patched font installed, you can override or remove any icon. Check the documentation below on the options available.
+![Frappe Flavor Preview](./assets/frappe.webp)
 
-> [!NOTE]
-> If you are using a tmux version older than 3.0, you
-> will need to follow the instructions for [older versions](#for-tmux-versions-prior-to-32).
+</details>
+<details>
+<summary>🌺 Macchiato</summary>
+
+![Macchiato Flavor Preview](./assets/macchiato.webp)
+
+</details>
+<details>
+<summary>🌿 Mocha</summary>
+
+![Mocha Flavor Preview](./assets/mocha.webp)
+
+</details>
+
+## Installation
+
+In order to have the icons displayed correctly please use/update your favorite
+ [nerd font](https://www.nerdfonts.com/font-downloads).
+If you do not have a patched font installed, you can override or remove any
+icon. Check the documentation on the options available.
```

Comment:
Is this referring to a specific document inside `docs/`? If so then I feel like we should link to it.